File: SEM 3/OOPL/LAB2/q1.java
import java.util.Scanner;

class q1
{
    public static void main(String args[])
    {
        int i,j,n,temp;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number of Elements : ");
        n=sc.nextInt();
        int a[]=new int[n];
        for(i=0;i<n;i++)
        {
            System.out.println("Enter Element "+(i+1)+" : ");
            a[i]=sc.nextInt();
        }
        for(i=0;i<n;i++)
        {
            for(j=0;j<n-i-1;j++)
            {
                if(a[j]>a[j+1])
                {
                    temp=a[j];
                    a[j]=a[j+1];
                    a[j+1]=temp;
                }
            }
        }
        System.out.println("Ascending Order is ");
        for(i=0;i<n;i++)
        {
            System.out.println(a[i]);
        }
        System.out.println("Descending Order is ");
        for(i=n-1;i>=0;i--)
        {
            System.out.println(a[i]);
        }
    }
}

File: SEM 3/DSAL/LAB1/q1.c
#include <stdio.h>

double Add(double a[],int n)
{
	int i;
	double sum=0;
	for(i=0;i<n;i++)
	{
		sum=sum+a[i];
	}
	return sum;
}

int main()
{
	int i,n;
	double a[100],sum;


	printf("Enter Number of Elements : \t");
	scanf("%d",&n);

	for(i=0;i<n;i++)
	{
		printf("Enter element %d : \t",i+1);
		scanf("%lf",&a[i]);
	}
	sum=Add(a,n);
	printf("Sum is %lf",sum);
	return 0;
}